src/flynt/cli.py

```diff
@@ -172,12 +173,18 @@ def run_flynt_cli(arglist: Optional[List[str]] = None) -> int:
         level=(logging.DEBUG if args.verbose else logging.CRITICAL),
     )
 
+    state = State(
```

Comment:
I think it's more usual to call such a shared object with settings a "Context".

Reply:
I wanted to keep the old name for symmetry's sake. It could be renamed in a future PR if that's necessary?

Reply:
alright.